tests/integration/devfile/cmd_devfile_describe_test.go

```diff
@@ -82,4 +83,25 @@ var _ = Describe("odo devfile describe command tests", func() {
 			helper.CmdShouldPass("odo", "delete", "-f", "--all", "--context", commonVar.Context)
 		})
 	})
+
+	Context("when running odo describe", func() {
```

Comment:
I think we should mention something like `for machine readable output` in the context.

Reply:
done